1.3/Defs/Morphs/Animal/Avian/Avian_Parts.xml

```diff
@@ -1,22 +1,99 @@
 <Defs xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../../../../Schemas/MutationDef.xsd">
 	<Pawnmorph.Hediffs.MutationDef Name ="AvianJawPart" ParentName="AnimalJawPart" Abstract="true">
-		<description>A morph beak.</description>
 		<mutationMemory>EtherBeak</mutationMemory>
+		<comps>
+			<li Class="HediffCompProperties_VerbGiver">
+				<tools>
+					<li>
+						<label>beak</label>
+						<labelUsedInLogging>False</labelUsedInLogging>
+						<capacities>
+							<li>Bite</li>
+						</capacities>
+						<power>10.0</power>
```

Comment:
I chose to give the verb definitions power 0 to indicate that the power is defined elsewhere, but as long as all stages have override defined, it doesn't matter.